worlds/pokemon_rb/level_scaling.py

```diff
@@ -137,3 +138,4 @@ def reachable():
                     sphere_objects[object].level = level_list_copy.pop(0)
     for world in multiworld.get_game_worlds("Pokemon Red and Blue"):
         world.finished_level_scaling.set()
+        
```

Comment:
My bad on the previous suggestion, the site added spaces and I missed them. This removes them for a proper EoF newline
```suggestion

```

Reply:
+1